ext/SpeedyWeatherCUDAExt/legendre.jl

```diff
@@ -0,0 +1,127 @@
+# convert i, j indices of a matrix (here 0-based l,m though...) to a single 1-based running index
+import LowerTriangularMatrices: ij2k
+
+# range of the running indices lm in a l-column (degrees of spherical harmonics)
+# given the column index m (order of harmonics) 
+get_lm_range(m, lmax) = ij2k(m, m, lmax):ij2k(lmax, m, lmax)
```

Comment:
This function assumes 1-based m, lmax inputs. I uses the `ij2k` function from LowerTriangularMatrices to transform between i, j indices of a matrix (here l, m) to the running index `k` as it's called in that module (here lm).